src/main/java/com/exasol/adapter/json/RequestJsonParser.java

```diff
@@ -62,7 +62,7 @@ public AdapterRequest parseRequest(String json) throws Exception {
             if (root.containsKey("pushdownRequest")) {
                 pushdownExp = root.getJsonObject("pushdownRequest");
             } else {
-                pushdownExp = root.getJsonObject("pushdownInquiry");    // This is outdated, remove when old versions are no longer used
+                throw new IllegalArgumentException("Push down expression was not identified.");
```

Comment:
Please be more specific. Suggest:
 "Push-down statement missing in adapter request element '/pushdowRequest'."